spec/requests/review_pages_spec.rb

```diff
@@ -255,6 +255,18 @@
          review.reload
          review.feedback_deadline.to_s(:short_date).should == Date.parse(new_deadline).to_s(:short_date)
          page.should have_selector('.flash-success',
+          text: 'Review was updated. Your coach and invitees have been notified of the new deadline.')
+       end
+
+       it "updates the review date" do
+         new_date = (review.review_date + 2.weeks).to_date.to_s
+         fill_in "review_review_date", with: new_date
+
+         click_button "Save Changes"
+
+         current_path.should == review_path(review)
+         review.reload
+         page.should have_selector('.flash-success',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.